oneview-ansible.md

```diff
@@ -1225,56 +1225,11 @@ Retrieve facts about the OneView Alerts.
 
 | Parameter     | Required    | Default  | Choices    | Comments |
 | ------------- |-------------| ---------|----------- |--------- |
-| config  |   No  |  | |  Path to a .json configuration file containing the OneView client configuration. The configuration file is optional. If the file path is not provided, the configuration will be loaded from environment variables.  |
```

Comment:
Please take a look in the doc generated, there are options, examples, and returned values missing.